tapir/translations/locale/de/LC_MESSAGES/django.po

```diff
@@ -572,39 +570,73 @@ msgstr "Der Betrag kann nicht negative sein"
 msgid "Amount paid for a share can't more than {COOP_SHARE_PRICE} (the price of a share)"
 msgstr "Der Betrag kann nicht mehr als {COOP_SHARE_PRICE} sein"
 
-#: coop/models.py:441
+#: coop/models.py:449
 msgid "Investing member"
 msgstr "Investierendes Mitglied"
 
-#: coop/models.py:448
+#: coop/models.py:456
 msgid "Signed Beteiligungserklärung"
 msgstr "Beteiligungserklärung unterschrieben"
 
-#: coop/models.py:451 coop/templates/coop/draftuser_detail.html:122
+#: coop/models.py:459 coop/templates/coop/draftuser_detail.html:123
 msgid "Paid Shares"
 msgstr "Anteil(e) bezahlt"
 
-#: coop/models.py:495
+#: coop/models.py:516
+#, fuzzy
+#| msgid "Email address"
+msgid "Email address must be set."
+msgstr "E-Mail-Adresse"
+
+#: coop/models.py:518
+#, fuzzy
+#| msgid "start date must be set"
+msgid "First name must be set."
```

Comment:
I think it says that the new string is similar to another one, so that editor programs can show suggestions. 

You can try to open the .po file with [poedit](https://poedit.net/).
Ideally we'd have an online tool for that, so that we can delegate some of the translation work to members, but we haven't set that up yet : https://github.com/SuperCoopBerlin/tapir/issues/412

Reply:
I think .po files are not intended to be edited manually.

Reply:
Right, that makes sense to not edit them manually. Though, what's the approach if one has to resolve merging conflicts?

Reply:
As far as I know, there's no good solution for them. I tend to do 

1. merge master in your branch, taking master's version of the .po file
2. generate the .po file with the merged version
3. translate
4. merge as fast as possible in master

but the quick search I did a while back didn't suggest a proper method.